tests/integration/test_integration.py

```diff
@@ -217,7 +217,7 @@ def test_single_trace_too_large():
     t = Tracer()
     with mock.patch("ddtrace.internal.writer.log") as log:
         with t.trace("huge"):
-            for i in range(100000):
+            for i in range(200000):
```

Comment:
This change was required because v0.5 produces smaller payloads so we need bigger traces to fill the buffer up.